### DISCUSSION TITLE
Version 0.17.5 still showing in GUI after updating to 0.18.2

### DISCUSSION THREAD

**Discussion by coryarobbins** [amidaware/tacticalrmm#1855] [Q&A]
Server Info (please complete the following information):

OS: Ubuntu 22.04.3 LTS
RMM Version (as shown in top left of web UI): 0.17.5
Installation Method:  Standard

Hello,
I've run the update script to go from v0.17.5 to current, but the version is still reported as 0.17.5 in the GUI.  I tried to re-run the script, but it reports as being the latest version.  I used the --force option, and no errors were reported, but the GUI still shows as v0.17.5. 

I've rebooted the server, completed all updates, and rerun the update script 3 times with --force.  I'm assuming I'm missing something, but I can't find it in the documentation.

**Comment by silversword411** [comment]
If you'd like us to diagnose your server please open a ticket.
https://docs.tacticalrmm.com/support/#commercial-support
